Extend/Email.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+namespace Wirecard\Oxid\Extend;
+
+/**
+ * @inheritdoc
+ */
+class Email extends Email_parent
+{
+    /**
+     * template for support email
+     *
+     * @var string
+     */
+    private $_sSupportEmailTemplate = 'module_support_email.tpl';
+
+    /**
+     * Send support email
+     *
+     * @param array $aEmailData Email data
+     *
+     * @return bool
+     */
+    public function sendSupportEmail($aEmailData)
+    {
+        $this->_clearMailer();
+        $oShop = $this->_getShop();
+        $oSmarty = $this->_getSmarty();
+
+        $this->setViewData('emailData', $aEmailData);
+        $this->setViewData('shopTemplateDir', $this->getConfig()->getTemplateDir(false));
+
+        $this->_processViewArray();
```

Comment:
Pleas refactor into logical methods as prepareMailer(), prepareEmailTemplate(), setEmailContent.

Reply:
The code above follows exactly the oxid email send function pattern, where the mailer settings vary per each send function. It would lead to these logical submethods for all the send mail function.